contracts/satelites/Power.sol

```diff
@@ -31,10 +30,14 @@ contract Power is Ownable, ERC20Basic {
   // ########### ADMIN FUNCTIONS ################
   // ############################################
 
-  function slashPower(address _holder, uint256 _value, bytes32 _data) onlyOwner {
+  function slashPower(address _holder, uint256 _value, bytes32 _data) public onlyOwner {
     Slashing(_holder, _value, _data);
   }
 
+  function powerUp(address _holder, uint256 _value) public onlyOwner {
+    // NTZ transfered from user's balance to power pool
+    Transfer(0x0, _holder, _value);
```

Comment:
yeah, i've been thinking about that a lot. advantage of using erc20 event:
- displayed in wallet history, when connecting to external wallets, like myEtherWallet
- recognized in etherscan, and i think etherscan only displays balances for accounts that were involved in events
disadvantage:
- cryptic
- waste of memory in event

i don't know, how to decide?